linux/CGroupUtils.c

```diff
@@ -357,3 +357,144 @@ char* CGroup_filterName(const char* cgroup) {
    s.buf[s.size] = '\0';
    return s.buf;
 }
+
+static bool CGroup_filterContainer_internal(const char* cgroup, StrBuf_state* s, StrBuf_putc_t w) {
+   const char* str_lxc_payload_legacy = "lxc.payload";
+   const char* str_lxc_payload_prefix = "lxc.payload.";
+
+   const char* str_nspawn_scope_prefix = "machine-";
+   const char* str_nspawn_monitor_label = "/supervisor";
+   const char* str_nspawn_payload_label = "/payload";
+
+   const char* str_pod_scope_prefix = "libpod-";
+
+   const char* str_scope_suffix = ".scope";
```

Comment:
@BenBE should these (above 10 lines) all be 'static const char*' so they're not on the stack?  I don't see them being modified but maybe I missed it.